tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2017",
-    "rootDir": "src",
+    "rootDir": ".",
```

Comment:
why is this needed ?

Reply:
because otherwise i get an error in api.ts when trying to import from "../../generated/definitions/api/requestTypes" (file is not under rootDir, rootDir is supposed to contain all source files